packages/react-components/react-progress/src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -47,4 +53,27 @@ describe('ProgressBar', () => {
     expect(result.getByRole('progressbar').getAttribute('aria-valuemin')).toEqual('0');
     expect(result.getByRole('progressbar').getAttribute('aria-valuemax')).toEqual('1');
   });
+  describe('Max', () => {
+    it('gives the proper error message when max is negative', () => {
+      const max = -1;
+      const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+      render(<ProgressBar max={max} />);
+      expect(console.error).toHaveBeenCalledWith(errorMsg);
+    });
+    it('gives the proper error message when max is zero', () => {
+      const max = 0;
+      const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+      render(<ProgressBar max={max} />);
+      expect(console.error).toHaveBeenCalledWith(errorMsg);
+    });
+  });
+  describe('Value', () => {
+    it('gives the proper error message when value is greater than max', () => {
+      const value = 23;
+      const max = 10;
+      const errorMsg = `The prop 'value' must be less than or equal to 'max'. Received  value: ${value}, max: ${max}`;
+      render(<ProgressBar value={value} max={max} />);
```

Comment:
Because `ProgressBar` will internally change values such that value and max are valid this test should also assert that `aria-valuemin/max/now` have the correct values set.

Reply:
The `aria-values` actually only get set when the `ProgressBar` is determinate, which will only happen when `value` is set, so I need to update the tests to reflect this